src/gcp/constants.ts

```diff
@@ -0,0 +1,13 @@
+import { AUTH_BLOCKING_EVENTS } from "../functions/events/v1";
+
+export const CODEBASE_LABEL = "firebase-functions-codebase";
+export const HASH_LABEL = "firebase-functions-hash";
+export const BLOCKING_LABEL = "deployment-blocking";
+export const BLOCKING_LABEL_KEY_TO_EVENT: Record<string, typeof AUTH_BLOCKING_EVENTS[number]> = {
+  "before-create": "providers/cloud.auth/eventTypes/user.beforeCreate",
+  "before-sign-in": "providers/cloud.auth/eventTypes/user.beforeSignIn",
+};
+export const BLOCKING_EVENT_TO_LABEL_KEY: Record<typeof AUTH_BLOCKING_EVENTS[number], string> = {
+  "providers/cloud.auth/eventTypes/user.beforeCreate": "before-create",
+  "providers/cloud.auth/eventTypes/user.beforeSignIn": "before-sign-in",
+};
```

Comment:
A downside for putting these here is that this directory is for all GCP services, not just functions. Maybe defining them in `src/functions` would be a better option. But @taeold might have some opinions about yanking these

Reply:
Agree with @colerogers here. Technically `gcp/cloudfunctions.ts` doesn't feel that great either, but at least that file is mainly used by us.